model/patch_lifecycle.go

```diff
@@ -1205,6 +1206,14 @@ func restartDiffItem(p patch.Patch, cq *commitqueue.CommitQueue) error {
 		PatchNumber:     patchNumber,
 	}
 
+	// The parser project is typically inserted at the same time as the patch.
```

Comment:
was duplicating this comment intentional?

Reply:
Hmm, it applies to two different use cases in the code (one for CLI merge patches, one for admin commit queue restarts), so I'd say it's equally important in both places.

Reply:
sounds good!